client/components/calendar/calendar.jsx

```diff
@@ -0,0 +1,89 @@
+import React, {Component} from 'react';
+import Styles from './styles.js';
+import moment from 'moment';
+
+import Week from './week.jsx';
+
+class Calendar extends Component {
+  constructor(props){
+    super(props);
+
+    this.state = {
+      date: moment(),
+      weeks: []
+    }
+
+    this.getDate = this.getDate.bind(this);
+    this.renderWeeks = this.renderWeeks.bind(this);
+  }
+
+  componentDidMount(){
+    this.renderWeeks();
+  }
+
+  getDate() {
+      console.log(moment().month(3).daysInMonth());
+  }
+
+  renderWeeks() {
+    let month = [];
+    let startOfMonthDay = moment().startOf('month').format('d');
+    let daysInMonth = moment().daysInMonth();
+
+    //create first array with appropriate amount of empty strings
+    let week = [];
+
+    for(let i=0; i<startOfMonthDay; i++){
+      week.push('');
+    }
+
+    for(let i=1; i<=7-startOfMonthDay; i++){
+      week.push(i);
+    }
+
+    month.push(week);
+
+    let count = 0;
+    week = [];
+
+    for(let i=8-startOfMonthDay; i<=daysInMonth; i++){
```

Comment:
You could fix the spacing in your for loops.

Reply:
Thanks for the tip!